services/dy-3dvis/docker-compose.yml

```diff
@@ -16,22 +16,22 @@ services:
         io.simcore.name: '{"name": "3d-viewer"}'
         io.simcore.description: '{"description": "Paraview Web-based Visualizer"}'
         io.simcore.authors: '{"authors": [
-                              {"name": "Sylvain Anderegg", "email": "anderegg@itis.swiss", 
+                              {"name": "Sylvain Anderegg", "email": "anderegg@itis.swiss",
                               "affiliation": "ITIS Foundation"}
                             ]}'
         io.simcore.contact: '{"contact": "anderegg@itis.swiss"}'
         io.simcore.inputs: '{"inputs": {
-                              "A": 
-                                {"displayOrder":1, "label": "1 dataset or n dataset in a zip", 
-                                "description": "Any paraview compatible dataset. Any number can be compressed in a zip file.", 
-                                "type": "data:*/*"}, 
-                              "B":{"displayOrder":2, "label": "1 dataset or n dataset in a zip", 
-                              "description": "Any paraview compatible dataset. Any number can be compressed in a zip file.", 
+                              "A":
+                                {"displayOrder":1, "label": "1 dataset or n dataset in a zip",
+                                "description": "Any paraview compatible dataset. Any number can be compressed in a zip file.",
+                                "type": "data:*/*"},
+                              "B":{"displayOrder":2, "label": "1 dataset or n dataset in a zip",
+                              "description": "Any paraview compatible dataset. Any number can be compressed in a zip file.",
```

Comment:
"Any number can be..." I guess you mean "an arbitrary number of files can be added as a compressed zip..."
Descriptions now end up in a tool-tip

Reply:
good point